lib/Middleware/WOPIMiddleware.php

```diff
@@ -97,6 +97,12 @@ public function afterException($controller, $methodName, \Exception $exception):
 			return new JSONResponse([], Http::STATUS_FORBIDDEN);
 		}
 
+		if ($exception instanceof \Exception) {
```

Comment:
As I see, this condition is always true. So, I guess we just need to return here.

Reply:
Good catch, I changed the condition slightly to only catch and return json if we actually want that for the WopiController